geoportailv3/static/js/catalog/layertreenode.html

```diff
@@ -1,14 +1,14 @@
 <a data-toggle="collapse" data-parent="#catalog-{{::layertreenodeCtrl.parentUid}}"
    href="#catalog-{{::layertreenodeCtrl.uid}}"
    ng-if="::layertreenodeCtrl.node.children && layertreenodeCtrl.depth < 3">
-  {{::layertreenodeCtrl.node.name}}
+  {{::layertreenodeCtrl.node.name | translate}}
```

Comment:
Using the `translate` filter should be reserved to cases where using the `translate` directive is not possible. See https://angular-gettext.rocketeer.be/dev-guide/annotate/. In that case I think we can use the `translate` directive.